Remotion/Core/UnitTests/Utilities/DateOnlyConverterTest.cs

```diff
@@ -0,0 +1,168 @@
+// SPDX-FileCopyrightText: (c) RUBICON IT GmbH, www.rubicon.eu
+// SPDX-License-Identifier: LGPL-2.1-or-later
+using System;
+using System.ComponentModel;
+using System.Globalization;
+using Moq;
+using NUnit.Framework;
+
+namespace Remotion.UnitTests.Utilities;
+
+using DateOnlyConverter = Remotion.Utilities.DateOnlyConverter;
+
+[TestFixture]
+public class DateOnlyConverterTest
+{
+  [Test]
+  public void CanConvertFrom_DateTime_ReturnsTrue ()
+  {
+    var converter = new DateOnlyConverter();
+
+    var result = converter.CanConvertFrom(Mock.Of<ITypeDescriptorContext>(), typeof(DateTime));
+
+    Assert.That(result, Is.True);
+  }
+
+  [Test]
+  public void CanConvertFrom_NullableDateTime_ReturnsTrue ()
+  {
+    var converter = new DateOnlyConverter();
+
+    var result = converter.CanConvertFrom(Mock.Of<ITypeDescriptorContext>(), typeof(DateTime?));
+
+    Assert.That(result, Is.True);
+  }
+
+  [Test]
+  public void CanConvertTo_DateTime_ReturnsTrue ()
+  {
+    var converter = new DateOnlyConverter();
+
+    var result = converter.CanConvertTo(Mock.Of<ITypeDescriptorContext>(), typeof(DateTime));
+
+    Assert.That(result, Is.True);
+  }
+
+  [Test]
+  public void CanConvertTo_NullableDateTime_ReturnsTrue ()
+  {
+    var converter = new DateOnlyConverter();
+
+    var result = converter.CanConvertTo(Mock.Of<ITypeDescriptorContext>(), typeof(DateTime?));
+
+    Assert.That(result, Is.True);
+  }
```

Comment:
I'm missing false-cases for DateOnly (should be explicitly stated here, I think) and possibly String (not sure), and some random type (for compelteness.